common/lib/capa/capa/templates/jsinput.html

```diff
@@ -43,9 +43,9 @@
   <br/>
     <p id="answer_${id}" class="answer"></p>
 
-    <p class="status">
-      <%include file="status_span.html" args="status=status, status_id=id"/>
-    </p>
+    <div class="indicator-container">
```

Comment:
Note that this problem type is only working due to the temporary commit that I added. Hopefully I will be able to update the version of JSME soon.